oncotator/oncotator-index.py

```diff
@@ -121,7 +123,7 @@
             out.write('\t'.join([tsv_data[h] for h in headers]) + '\n')
         out.close()
 
-        pysam.tabix_index(filename=output_fname, seq_col=15, start_col=16, end_col=17)
+        pysam.tabix_index(filename=output_fname, seq_col=34, start_col=35, end_col=36)
```

Comment:
are these dependent on cosmic version?  should there be a check if version is outside of the known valid ranges for these parameters?